### PR TITLE
add purge options

### PR DESCRIPTION
**Pull request by famarting**
# Description

blocked by https://github.com/microsoft/durabletask-go/pull/82

closes https://github.com/dapr/go-sdk/issues/497

## Issue reference

Please reference the issue this PR will close: #_[issue number]_

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Extended the documentation

### CODE REVIEW

File: workflow/client.go [hunk 1]
```diff
@@ -205,9 +215,10 @@ func (c *Client) ResumeWorkflow(ctx context.Context, id, reason string) error {
 
 // PurgeWorkflow will purge a given workflow and return an error output.
 // NOTE: The workflow must be in a terminated or completed state.
-func (c *Client) PurgeWorkflow(ctx context.Context, id string) error {
+// TODO missing upstream support for api.PurgeOptions
```
Comment (mikeee) [inline workflow/client.go:218]: Looks like this comment can be removed with the merge of your PR upstream

**Review by mikeee — APPROVED** [review]
lgtm - thanks for making the changes

**Comment by famarting** [comment]
merge please

**Comment by mikeee** [comment]
@yaron2 Are you happy to merge?

**Pull request merged**